source/MRViewer/MRRibbonNotification.cpp

```diff
@@ -209,7 +407,23 @@ void RibbonNotifier::drawNotifications( float scaling )
         ImGui::PopStyleVar( 4 );
         currentPos.y -= window->Size.y;
     }
-    filterInvalid_( numInvalid );
+
+    historyBtnPosY_ = currentPos.y - cWIndowSpacing * scaling;
```

Comment:
why do we update it each frame? should be always in the same position?

Reply:
yes, button is located above upper-left corner of notifications